integration_tests/tests/forseti/controls/inventory.rb

```diff
@@ -0,0 +1,115 @@
+            end
+
+            it "should be visible in the database" do
+                expect(command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"select COUNT(DISTINCT gcp_inventory.inventory_index_id) from gcp_inventory join inventory_index ON inventory_index.id = gcp_inventory.inventory_index_id;\"").stdout).to match /1/
+            end
+
+             it "should be visible in the database" do
+                 expect(command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"SELECT count(DISTINCT resource_data->>'$.lifecycleState') FROM gcp_inventory WHERE category = 'resource' and resource_type = 'project' and resource_data->>'$.lifecycleState' = 'ACTIVE';\"").stdout).to match /1/
+             end
+
+             it "should be visible in the database" do
+                 expect(command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"SELECT count(DISTINCT resource_type) from gcp_inventory where resource_type in ('kms_cryptokey', 'kms_keyring');\"").stdout).to match /#{resources_names.count}/
+             end
+        end
+
+        describe "List an inventory" do
+
+            it "should be visible from the command-line" do
+                expect(command("forseti inventory list").stderr).to eq ""
+            end
+
+            it "should be visible in the database" do
+                expect(command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"select count(DISTINCT gcp_inventory.inventory_index_id) from gcp_inventory join inventory_index on inventory_index.id = gcp_inventory.inventory_index_id where inventory_index.id = #{@inventory_id};\"").stdout).to match /1/
+            end
+        end
+
+        describe "Get an inventory" do
+
+            it "should be visible from the command-line" do
+                expect(command("forseti inventory get #{@inventory_id}").stderr).to eq ""
+            end
+
+            it "should be visible in the database" do
+                expect(command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"select * from inventory_index where id = #{@inventory_id};\"").stdout).to match /1/
+            end
+        end
+
+        describe "Delete an inventory" do
+
+            before :context do
+                command("forseti inventory create").result
+            end
+
+            # Displays number of inventories in the database before purging inventories.
+            it "should be visible in the database" do
+                expect(command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"select count(DISTINCT gcp_inventory.inventory_index_id) from gcp_inventory join inventory_index ON inventory_index.id = gcp_inventory.inventory_index_id;\"").stdout).to match /2/
+            end
+
+            it "should be visible from the command-line" do
+                expect(command("forseti inventory delete #{@inventory_id}").stderr).to match ""
```

Comment:
ditto, should check on `stdout`

Reply:
Done

Reply:
This still checking on `stderr`.

Reply:
I thought it would be nice to check on both `stdout` and `stderr`. Removed `stderr` checks as requested.